security/ldap.rst

```diff
@@ -268,6 +270,14 @@ implementation. Commonly used values are:
 If you pass ``null`` as the value of this option, the default UID key is used
 ``sAMAccountName``.
 
+extra_fields
+............
+
+**type**: ``array`` **default**: ``null``
+
+This is the custom fields to pull from the LDAP server.
+
```

Comment:
Maybe a note on how it behaves if an extra field isn’t available? Will it be null or does it throw an exception?